docs/content/developer/web-api-interaction.md

```diff
@@ -0,0 +1,39 @@
+# Web API Interaction { #android_sdk_web_api_interaction }
+
+When using the SDK sometimes one would like to make requests that are not supported by the default sdk, but fortunately the SDK offers another approach allowing you to interact with the Web API and here are the steps to follow.
+
+## Retrofit Service
+
+The SDK uses Retrofit to make requests to the Web API and the first thing we have to do is to create a retrofit service by specifying a particular end-point. Everything you already known about [Retrofit](https://square.github.io/retrofit/) Can be more or less used here.
```

Comment:
There is a type here? Should it be "Everything you already **know** about ..". And the word "Can" should be lowercase.